include/swift/AST/DiagnosticsSema.def

```diff
@@ -4493,8 +4493,8 @@ ERROR(actor_isolated_non_self_reference,none,
         "from the main actor|from a non-isolated context}3",
         (DescriptiveDeclKind, DeclName, unsigned, unsigned, Type))
 ERROR(distributed_actor_isolated_non_self_reference,none,
-      "distributed actor-isolated %0 %1 can only be referenced inside the "
-      "distributed actor",
+      "distributed actor-isolated %0 %1 can not be accessed from a "
+      "non-isolated context",
```

Comment:
These try to be more consistent with the rest of the current concurrency messages

Reply:
Nit: `cannot` and please drop `a` before `non-isolated context"

Reply:
Or `cannot access distributed actor-isolated %0 %1 from non-isolated context` even which is more inline with other diagnostics we have in Sema.

Reply:
The message this tries to be same-as is:

```
ERROR(actor_isolated_non_self_reference,none,
        "actor-isolated %0 %1 can not be "
        "%select{referenced|mutated|used 'inout'}2 "
        "%select{on a non-isolated actor instance|"
        "from a Sendable function|from a Sendable closure|"
        "from an 'async let' initializer|from global actor %4|"
        "from the main actor|from a non-isolated context}3",
        (DescriptiveDeclKind, DeclName, unsigned, unsigned, Type))
```
So it is `actor-isolated %0 %1 can not be referenced from a non-isolated context`

Reply:
Ok will adjust :) and eventually the concurrency ones